tests/CMakeLists.txt

```diff
@@ -387,6 +387,16 @@ target_link_libraries(detect_encodings_tests
   PUBLIC simdutf::tests::helpers
          simdutf::tests::reference)
 
+add_cpp_test(text_encoding_tests)
+target_link_libraries(text_encoding_tests
+  PUBLIC simdutf::tests::helpers
+         simdutf::tests::reference)
+         
+if(cxx_std_26 IN_LIST CMAKE_CXX_COMPILE_FEATURES)
+  set_property(TARGET text_encoding_tests PROPERTY CXX_STANDARD 26)
+  set_property(TARGET text_encoding_tests PROPERTY CXX_STANDARD_REQUIRED OFF)
+endif()
+
```

Comment:
One thing the unit tests are there to check is that it is possible to compile the project with that specific standard. This would unconditionally enable C++26 if the compiler supports it. So I would like to see these lines removed.

Instead, I suggest adding a specific C++26 build to the ci jobs. Can you add a new ci job which uses gcc 15?

Reply:
closing this comment, a C++26 job has been merged on the master branch.